shared/src/packages/sites/libs/types/sites-get-all-parameters-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type SitesGetAllParametersDto = {
```

Comment:
not sure that you use this type

Reply:
i did it like in Search Projects,  for me it strange why do we need two different types, one on the page and other in slices and backend.

Reply:
`/projects/:id`  – parameters
`/projects?name=supper` – query parameters

Reply:
http://localhost:3000/api/v1/sites/project/2?name=vla here is our search string, 
/projects?name=supper – query parameters we need to find not from projects sites but all sites?

Reply:
or you mean to crate first class for ID, scond for searc parameters and third to connect it in action?